### PR TITLE
breadth, categories, and weights added to scraper

### PR DESCRIPTION
**Pull request by ArsalaanAli**
scraping and uploading course weights and categories to db

### CODE REVIEW

File: cat_data.csv [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+104,PHYSIOL,['C']
+105,PHYSPHRM,['C']
+106,POLISCI,['A']
+107,PPE,['A']
+108,PORTUGSE,['B']
+109,PSYCHOL,['A']
+110,REHABSCI,['A']
+111,RELSTUD,['B']
+112,RUSSIAN,['B']
+113,SCHOLARS,[]
```
Comment (basokant) [inline cat_data.csv]: How are we handling this category?

File: databaseFunctions.py [hunk 2]
```diff
@@ -46,21 +57,34 @@ def addJsonToTable(cursor, conn, courseCode, columnName, data):
     conn.commit()
 
 
-def insertCourseIntoDatabase(name, code, prereqs, antireqs, coreqs, precoreqs, desc, location, extra, conn, cursor):
+def insertCategoryIntoDatabase(category, breadth, conn, cursor):
+    insertCatQuery = "INSERT INTO \"Category\" (category_name, breadth) VALUES (%s, %s)"
+    cursor.execute(insertCatQuery, (category, breadth))
```
Comment (basokant) [inline databaseFunctions.py]: Shoot I didn't mention this in the fe pr but could we store the full category name here as well?

**Review by basokant — COMMENTED** [review]
Just one comment. I think we should store the full category name so we can include it somewhere on a course page. What do you think? Other than that the code looks good to me!

File: databaseFunctions.py (hunk 2 above)
Comment (ArsalaanAli) [reply databaseFunctions.py]: oh good idea, yea ill do that

**Pull request merged**